code-challenges.test.js

```diff
@@ -41,14 +50,23 @@ var cubeAndSum1 = [2, 3, 4]
 // Expected output: 99
 var cubeAndSum2 = [0, 5, 10]
 // Expected output: 1125
-
-
+const numsCubed = (array) => {
+  let numbers = []
+  for (var i = 0; i < array.length; i++) {
+    array[i] * 2
```

Comment:
almost there! 
```suggestion
    numbers.push(array[i] * 2)
```

You need to make sure to store the cubed value into the new array you created.